ensure_tiller_installed.go

```diff
@@ -207,10 +208,18 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 
 			return nil
 		}
-		b := backoff.NewExponential(1*time.Minute, 5*time.Second)
+
+		// maxWait is configurable so we can use more time in e2e tests
+		// than in operators.
+		maxWait, err = time.ParseDuration(c.ensureTillerInstalledMaxWait)
```

Comment:
Considering this here is a library we can also configure the properly typed `time.Duration`. No need for the string magic and runtime parsing I think.

Reply:
It's dumb but I had trouble comparing the `time.Duration` to see if its set. If it's unset we need to use the 60 second default.

So that's why I did it this way. Any ideas?

Reply:
OK its 0 I'l get it fixed 😄

Reply:
Done and it's way nicer.